webapp/src/index.js

```diff
@@ -22,6 +23,9 @@ class PluginClass {
         registry.registerBottomTeamSidebarComponent(TeamSidebar);
         registry.registerPopoverUserAttributesComponent(UserAttribute);
 
+        const {showRHSPlugin} = registry.registerRightHandSidebarComponent(SidebarRight, 'Github Plugin');
+        store.dispatch(receivedRHSAction(showRHSPlugin));
```

Comment:
I think we can wrap this in dispatch here before saving it so we don't have that strangeness later. 
```suggestion
        store.dispatch(receivedRHSAction(() => store.dispatch(showRHSPlugin)));
```

Reply:
So much better! I tried so many different ways that I ended up in a weird combination of options